sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -745,7 +768,27 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
   ): Promise<HttpOperationResponse> {
     const webResource: WebResource = new WebResource(this.getUrl(name), "GET");
 
-    return executeAtomXmlOperation(this, webResource, serializer);
+    return new Promise<HttpOperationResponse>(async (resolve, reject) => {
```

Comment:
I dont understand why we need to create a `new Promise` here. Doesnt the below suffice?

```javascript
const response = await executeAtomXmlOperation(this, webResource, serializer);
if (your-checks-here-for-404-cases) {
     // throw the rest error
}
return response;
```

Reply:
When you say you don't understand, is it that this is a convention/practice that is never or not to be used?

Reply:
I think it's because wrapping it in a promise is redundant. Same reasoning as here:
https://github.com/Azure/azure-sdk-for-js/pull/6112#discussion_r345537077

Reply:
Oh, I thought I had already updated the same occurrence here - looks like some commits didnt get pushed.
PR should be updated now @ramya-rao-a @chradek